lib/splitclient-rb/cache/senders/events_sender.rb

```diff
@@ -0,0 +1,47 @@
+module SplitIoClient
+  module Cache
+    module Senders
+      class EventsSender
+        def initialize(events_repository, config, api_key)
+          @events_repository = events_repository
+          @config = config
+          @api_key = api_key
+        end
+
+        def call
+          if ENV['SPLITCLIENT_ENV'] == 'test'
+            post_events
+          else
+            events_thread
+
+            if defined?(PhusionPassenger)
+              PhusionPassenger.on_event(:starting_worker_process) do |forked|
```

Comment:
what's the story with puma server here?

Reply:
Same thing as with every other thread we have: we're recreating threads for Passenger workers